src/main/java/frc/robot/commands/TransitionRunCommand.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.constants.StorageConstants;
+import frc.robot.subsystems.Storage;
+
+public class TransitionRunCommand extends Command {
+    private Storage storage = Storage.getInstance();
+
+    private Timer timer = new Timer();
+    private double timeToRun = StorageConstants.defaultTimeToRun;
```

Comment:
Same comment about defaulting to 0 so you can have the infinite run case if a time is not specified